.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -5,6 +5,12 @@ title: 'Feature Request - [YOUR FEATURE REQUEST]'
 
 ---
 
+How to use this template:
+Any line that begins with '#' is a **header**. Please leave the headers in as is, and fill in the information they suggest
+below them. These templates are here to help us resolve your issue as quickly as possible. Leaving requested information out
+of your issue will cause our issue template bot to request that your post be reformatted. If a header doesn't seem relevant
+to your issue, either write 'N/A' or something similar or consider using a different template.
```

Comment:
You could add "either write 'N/A' or something similar below that header or consider using a different template" to make it more obvious that the header should stay.

Reply:
Good idea. I'll change that in the others ones as well.